src/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.java

```diff
@@ -25,26 +25,27 @@
         programGroup = Metrics.class
 )
 public class ConvertSequencingArtifactToOxoG extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Extract OxoG metrics from generalized artifacts metrics";
-    static final String USAGE_DETAILS = "This tool extracts 8-oxoguanine (OxoG) artifact metrics from the output of " +
+    static final String USAGE_SUMMARY = "Extract OxoG metrics from generalized artifacts metrics.  ";
+    static final String USAGE_DETAILS = "<p>This tool extracts 8-oxoguanine (OxoG) artifact metrics from the output of " +
             "CollectSequencingArtifactsMetrics(a tool that provides detailed information on a variety of artifacts found in sequencing " +
             "libraries) and converts them to the CollectOxoGMetrics tool's output format. This conveniently eliminates the need to run " +
             "CollectOxoGMetrics if we already ran CollectSequencingArtifactsMetrics in our pipeline. See the documentation for " +
-            "CollectSequencingArtifactsMetrics and CollectOxoGMetrics for additional information on these tools.<br /><br />." +
-            "Note that only the base of the CollectSequencingArtifactsMetrics output file name is required for the file name input. " +
+            "CollectSequencingArtifactsMetrics and CollectOxoGMetrics for additional information on these tools.</p>." +
+            "" +
+            "<p>Note that only the base of the CollectSequencingArtifactsMetrics output file name is required for the file name input. " +
             "For example, if the file name is artifact_metrics.txt.bait_bias_detail_metrics" +
-            " or artifact_metrics.txt.pre_adapter_detail_metrics, only the base of the file name \"artifact_metrics\" is " +
-            "required on the command line for \"input\".  " +
-            "An output file called \"artifact_metrics.oxog_metrics\" will be generated automatically.  " +
-            "A reference sequence is also required.<br />"+
+            " or artifact_metrics.txt.pre_adapter_detail_metrics, only the base of the file name 'artifact_metrics' is " +
+            "required on the command line for 'input'.  " +
+            "An output file called 'artifact_metrics.oxog_metrics' will be generated automatically.  " +
+            "A reference sequence is also required.</p>"+
```

Comment:
@dekling What this will also do it make it so that REFERENCE_SEQUENCE appears in the regular --help for this tool. currently it is hidden because it's annotation definition has `common=true`, but if you override the variable and annotation, you will not make it a common variable, and thus it will show up in the help...also in the online-help...I think it will actually be more helpful than text in the USAGE_DETAILS